src/aac/util.py

```diff
@@ -107,7 +113,9 @@ def search(model: dict[str, Any], search_keys: list[str]) -> list:
 
 def get_aac_spec() -> tuple[dict[str, dict], dict[str, dict]]:
     """
-    Gets the specification for Architecture-as-Code iteself.  The AaC model specification is
+    Get the specification for Architecture-as-Code itself.
+
+    The AaC model specification is
```

Comment:
This and the next line should be joined.

Reply:
This ties into our docstring discussions, but the intention was that first line should be a summary and the second line was the start of a more in-depth description.